src/components/Pricing/PricingExperiment.tsx

```diff
@@ -0,0 +1,1326 @@
+                                        </SidebarListItem>
+                                        <SidebarListItem>
+                                            We also offer{' '}
+                                            <Tooltip content={() => <Discounts />} placement="top">
+                                                <strong className="text-red dark:text-yellow border-b border-dashed border-light dark:border-dark cursor-help text-primary/75 dark:text-primary-dark/75">
+                                                    discounts
+                                                </strong>
+                                            </Tooltip>{' '}
+                                            for startups and non-profits
+                                        </SidebarListItem>
+                                    </SidebarList>
+                                </div>
+                                <div>
+                                    <h4 className="text-lg mb-2">Estimating usage</h4>
+                                    <SidebarList>
+                                        <SidebarListItem>
+                                            Not sure what your volume looks like? Add the tracking code to your site and
+                                            check back in a few days – no credit card required.
+                                        </SidebarListItem>
+                                        <SidebarListItem>
+                                            If something stupid happens, like you get an unexpected bill and you’re
+                                            unhappy, we’ll pretty much always refund it.
+                                        </SidebarListItem>
+                                    </SidebarList>
+                                </div>
+                            </SectionSidebar>
+                        </SectionColumns>
+                    </SectionLayout>
+
+                    <SectionLayout>
+                        <SectionHeader>
+                            <h3>One plan for most customers</h3>
+                        </SectionHeader>
+
+                        <SectionColumns>
+                            <SectionMainCol>
+                                <PlansTabs />
+                            </SectionMainCol>
+                            <SectionSidebar className="justify-between">
+                                <div>
+                                    <h4 className="text-lg mb-2">Plan FYIs</h4>
+                                    <SidebarList>
+                                        <SidebarListItem>
+                                            Self-serve, no upsells, no need to "talk to sales"
+                                        </SidebarListItem>
+                                        <SidebarListItem>
+                                            We don't do outbound sales. Everyone pays the same rates.
```

Comment:
Enterprise customers sometimes get better rates... I don't think we should say everyone pays the same rates.